src/unity/python/turicreate/data_structures/image.py

```diff
@@ -194,6 +194,10 @@ def _to_pil_image(self):
         from PIL import Image as _PIL_Image
         return _PIL_Image.fromarray(self.pixel_data)
 
+    def save(self, filename):
+        from PIL import Image as _PIL_Image
```

Comment:
To avoid duplicating the above code, call `self._to_pil_image().save(filename)`.

Reply:
I was considering that but I saw @gustavla 's comment at #340 where he said that it might break without warning at any new version and a more API-stable approach would be to do:
```
from PIL import Image
Image.fromarray(img.pixel_data).save('test.jpg')
```